abaco.go

```diff
@@ -129,26 +129,32 @@ func (group *AbacoGroup) updateFrameTiming(p *packets.Packet, frameIdx FrameInde
 	if ts == nil {
 		return
 	}
+	newSubframeCount := frameIdx * abacoSubsampleDivisions
+	deltaSubframe := newSubframeCount - group.LastSubframeCount
 	deltaTs := ts.T - group.LastTimestamp.T
-	if frameIdx <= group.LastRowCount || deltaTs == 0 {
+	// There will be transient nonsense if the timestamp.Rate changes. I think the best
+	// approach is set subframe rate to 0. After the next (consistent) timestamp, it will recover.
+	unequalRates := ts.Rate != group.LastTimestamp.Rate
+	group.LastTimestamp = *ts
+	group.LastSubframeCount = newSubframeCount
+	if deltaSubframe <= 0 || deltaTs == 0 || unequalRates {
```

Comment:
Wait are we allowing that the `SubframeRate` might change during acquisition? Or is this solely a learning step?

Reply:
I see this as entirely a learning step. But if it did change, this would catch it fast. 

What I really had in mind, though, is not a change in _rate_, but the possibility that somehow the _alignment_ between counters gets off (maybe a dropped packet could do that?). This would allow the two to re-synchronize quickly.

Reply:
But doesn't that mean there was data corruption? I guess we've decided to allow dropped packets so maybe then? Can we at least print something out if it changes so there is some breadcrumb.